modules/ding_gatewayf/ding_gatewayf.info

```diff
@@ -0,0 +1,6 @@
+name = Ding gatewayf
+description = Support for login at gatewayf.
```

Comment:
Could this be elaborated a bit?

There is no README and I see no explanatory DocBlock about what gatewayf actually is. Googling it does not help either.